data/mods/MindOverMatter/powers/photokinesis.json

```
@@ -0,0 +1,334 @@
+    "base_energy_cost": 2000,
+    "final_energy_cost": 500,
+    "energy_increment": -150,
+    "base_casting_time": 125,
+    "final_casting_time": 50,
+    "casting_time_increment": -7.5,
+    "learn_spells": { "photokinetic_light_arms": 7, "photokinetic_invisibility": 12 }
+  },
+  {
+    "id": "photokinetic_light_dodge",
+    "type": "SPELL",
+    "name": "[Ψ]Trick of the Light",
+    "description": "Manipulate the light around your body, creating small illusions and becoming harder to hit.",
+    "message": "",
+    "teachable": false,
+    "valid_targets": [ "self" ],
+    "spell_class": "PHOTOKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS", "NO_EXPLOSION_SFX", "RANDOM_DURATION" ],
+    "effect": "attack",
+    "effect_str": "effect_photokin_dodge",
+    "extra_effects": [ { "id": "psionic_drained_difficulty_two", "hit_self": true } ],
+    "shape": "blast",
+    "min_duration": {
+      "math": [
+        "( (u_val('spell_level', 'spell: photokinetic_light_dodge') * 2000) + 3000) * (scaling_factor(u_val('intelligence') ) )"
+      ]
+    },
+    "max_duration": {
+      "math": [
+        "( (u_val('spell_level', 'spell: photokinetic_light_dodge') * 2000) + 6000) * (scaling_factor(u_val('intelligence') ) )"
+      ]
+    },
+    "energy_source": "STAMINA",
+    "difficulty": 2,
+    "max_level": { "math": [ "int_to_level(1)" ] },
+    "base_energy_cost": 2500,
+    "final_energy_cost": 1000,
+    "energy_increment": -125,
+    "base_casting_time": 250,
+    "final_casting_time": 100,
+    "casting_time_increment": -12.5,
+    "learn_spells": { "photokinetic_rad_immunity": 5, "photokinetic_invisibility": 7, "photokinetic_light_image": 12 }
+  },
+  {
+    "id": "photokinetic_light_beam",
+    "type": "SPELL",
+    "name": "[Ψ]Photon Beam",
+    "description": "Concentrate photons into a high-energy beam.",
+    "message": "",
+    "teachable": false,
+    "valid_targets": [ "ground", "hostile" ],
+    "spell_class": "PHOTOKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "CONCENTRATE", "NO_HANDS", "NO_LEGS", "NO_EXPLOSION_SFX", "RANDOM_DAMAGE" ],
+    "effect": "attack",
+    "extra_effects": [ { "id": "psionic_drained_difficulty_two", "hit_self": true } ],
+    "shape": "line",
+    "min_damage": {
+      "math": [ "( (u_val('spell_level', 'spell: photokinetic_light_beam') * 2) + 3) * (scaling_factor(u_val('intelligence') ) )" ]
+    },
+    "max_damage": {
+      "math": [ "( (u_val('spell_level', 'spell: photokinetic_light_beam') * 2) + 15) * (scaling_factor(u_val('intelligence') ) )" ]
+    },
+    "damage_type": "psi_photokinetic_damage",
+    "min_range": {
+      "math": [ "( (u_val('spell_level', 'spell: photokinetic_light_beam') * 0.6) + 2) * (scaling_factor(u_val('intelligence') ) )" ]
+    },
+    "max_range": 30,
+    "energy_source": "STAMINA",
+    "difficulty": 2,
+    "max_level": { "math": [ "int_to_level(1)" ] },
```

```suggestion
    "max_level": { "math": [ "int_to_level(1)" ] },
    "field_id": "fd_laser",
    "min_field_intensity": 2,
    "max_field_intensity": 2,
    "field_chance": 1,
    
```
This will leave lasers in the path of the attack for a turn or two.